symfony/messenger/4.3/post-install.txt

```diff
@@ -0,0 +1,19 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * You're ready to use the Messenger component. You can define your own message buses
+    or start using the default one right now by injecting the <info>messenger.bus.default</info> service
+    or typehinting <info>Symfony\Component\Messenger\MessageBusInterface</info> in your code.
+
+  * If you need to send messages to your broker, you can benefit from the built-in
+    AMQP transport by:
+
+    1. Installing the AMQP pack by running <info>composer require amqp</>;
```

Comment:
Probably remove this - and add it as a "detail" in `.env` above the `amqp` example. We're making a lot of changes, so we'll need to play with the final result to see how it feels

Reply:
I wonder if we should not rename the pack too? Or depreciate it ?
Because it includes the serializer pack that is also useful for other transport...